form/2d/p5js/closethetriangle/index.js

```diff
@@ -0,0 +1,21 @@
+function setup() {
+  createCanvas (500,500);
```

Comment:
the canvas needs to be assigned to the sketch id div. See https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code